yarn.lock

```diff
@@ -0,0 +1,3497 @@
+
+safe-buffer@5.2.1, safe-buffer@^5.0.1, safe-buffer@^5.1.0, safe-buffer@^5.1.1, safe-buffer@^5.1.2, safe-buffer@^5.2.0, safe-buffer@~5.2.0:
+  version "5.2.1"
+  resolved "https://registry.yarnpkg.com/safe-buffer/-/safe-buffer-5.2.1.tgz#1eaf9fa9bdb1fdd4ec75f58f9cdb4e6b7827eec6"
+  integrity sha512-rp3So07KcdmmKbGvgaNxQSJr7bGVSVk5S9Eq1F+ppbRo70+YeaDxkw5Dd8NPN+GD6bjnYm2VuPuCXmpuYvmCXQ==
+
+safe-regex@^1.1.0:
+  version "1.1.0"
+  resolved "https://registry.yarnpkg.com/safe-regex/-/safe-regex-1.1.0.tgz#40a3669f3b077d1e943d44629e157dd48023bf2e"
+  integrity sha512-aJXcif4xnaNUzvUuC5gcb46oTS7zvg4jpMTnuqtrEPlR3vFr4pxtdTwaF1Qs3Enjn9HK+ZlwQui+a7z0SywIzg==
+  dependencies:
+    ret "~0.1.10"
+
+"safer-buffer@>= 2.1.2 < 3", safer-buffer@^2.1.0:
+  version "2.1.2"
+  resolved "https://registry.yarnpkg.com/safer-buffer/-/safer-buffer-2.1.2.tgz#44fa161b0187b9549dd84bb91802f9bd8385cd6a"
+  integrity sha512-YZo3K82SD7Riyi0E1EQPojLz7kpepnSQI9IyPbHHg1XXXevb5dJI7tpyN2ADxGcQbHG7vcyRHk0cbwqcQriUtg==
+
+saml-idp@^1.2.1:
+  version "1.2.1"
+  resolved "https://registry.yarnpkg.com/saml-idp/-/saml-idp-1.2.1.tgz#2df394cd406ce273641115f587062ea60d29ce03"
+  integrity sha512-C7iXTxryohn8fOWUGyPDCJwyA4eWyh4gJrMAndyRe+idTrj51kPOW1FAAsIBc7afmoJLM00qqcAP/gxalq4r9A==
+  dependencies:
+    body-parser "~1.19.0"
+    chalk "^4.0.0"
+    debug "~4.1.1"
+    express "^4.17.1"
+    express-session "^1.17.1"
+    extend "^3.0.2"
+    hbs "^4.1.1"
+    morgan "^1.10.0"
+    samlp "github:mcguinness/node-samlp"
+    xml-formatter "^2.1.0"
+    xmldom "^0.3.0"
+    yargs "^15.3.1"
+
+saml@^1.0.0:
+  version "1.0.1"
+  resolved "https://registry.yarnpkg.com/saml/-/saml-1.0.1.tgz#fdcd8b46bf59ce9ad1ec124e8061da2d7184d256"
+  integrity sha512-BzzlTdXNICrIGhJkq168n0WJpwXYr3xyMd7UHC7/s8F4M6zHSEItwEuKGmm6HjsttZk/hJcrw7fY0OZ9wE+v7Q==
+  dependencies:
+    "@xmldom/xmldom" "^0.7.4"
+    async "~0.2.9"
+    moment "2.19.3"
+    valid-url "~1.0.9"
+    xml-crypto "^2.1.3"
+    xml-encryption "^1.2.1"
+    xml-name-validator "~2.0.1"
+    xpath "0.0.5"
+
+samlp@6.0.2, "samlp@github:mcguinness/node-samlp":
+  version "6.0.2"
+  resolved "https://registry.yarnpkg.com/samlp/-/samlp-6.0.2.tgz#e0468f9ecddcd2f34c216ffcf0ff2e8c7e0fba35"
+  integrity sha512-LeEgzxBRRasDk4EOiGZgIa4ytvHNnYDMI9Wp2tGh+v7fi3bjZYG1ufJK3YWGDTb5HFG8Y7ybpYzw/o2tThBgLQ==
+  dependencies:
+    "@auth0/thumbprint" "0.0.6"
+    "@auth0/xmldom" "0.1.21"
+    auth0-id-generator "^0.2.0"
+    ejs "2.5.5"
+    flowstate "^0.4.0"
+    querystring "^0.2.0"
+    saml "^1.0.0"
+    xml-crypto "^2.0.0"
+    xpath "0.0.5"
+    xtend "^1.0.3"
+
+schema-utils@^1.0.0:
+  version "1.0.0"
+  resolved "https://registry.yarnpkg.com/schema-utils/-/schema-utils-1.0.0.tgz#0b79a93204d7b600d4b2850d1f66c2a34951c770"
+  integrity sha512-i27Mic4KovM/lnGsy8whRCHhc7VicJajAjTrYg11K9zfZXnYIt4k5F+kZkwjnrhKzLic/HLU4j11mjsz2G/75g==
+  dependencies:
+    ajv "^6.1.0"
+    ajv-errors "^1.0.0"
+    ajv-keywords "^3.1.0"
+
+selenium-webdriver@^4.0.0-alpha.7:
```

Comment:
This was manually added to `yarn.lock` to make sure there is a compatible version with node 10.24.1